src/main/steps/c100-rebuild/miam/mediator-document/content.test.ts

```diff
@@ -0,0 +1,54 @@
+import languageAssertions from '../../../../../test/unit/utils/languageAssertions';
+import { FormContent, FormFields, FormOptions, LanguageLookup } from '../../../../app/form/Form';
+import { CommonContent, generatePageContent } from '../../../common/common.content';
+
+import { generateContent } from './content';
+
+jest.mock('../../../../app/form/validation');
+
+const en = {
+  headingTitle: 'Do you have a document signed by the mediator?  ',
+  docSigned:
+    'The mediator should give you a signed document to confirm you attended a MIAM, or do not need to attend. If you do not have a document, you should ask the mediator for one.',
+  one: 'Yes',
+  two: 'No',
+};
+
+const cy = {
+  headingTitle: 'Do you have a document signed by the mediator? - welsh  ',
+  docSigned:
+    'The mediator should give you a signed document to confirm you attended a MIAM, or do not need to attend. If you do not have a document, you should ask the mediator for one. - Welsh',
+  one: 'Yes - Welsh',
+  two: 'No - Welsh',
+};
+
+describe('miam->have document signed by mediator or not', () => {
+  const commonContent = { language: 'en', userCase: { applyingWith: 'alone' } } as unknown as CommonContent;
+  // eslint-disable-next-line jest/expect-expect
+  test('should return correct english content', () => {
+    languageAssertions('en', en, () => generateContent(commonContent));
+  });
+
+  // eslint-disable-next-line jest/expect-expect
+  test('should return correct welsh content', () => {
+    languageAssertions('cy', cy, () => generateContent({ ...commonContent, language: 'cy' }));
+  });
+  test('should contain applyingWith field', () => {
+    const generatedContent = generateContent(commonContent) as Record<string, never>;
+    const form = generatedContent.form as FormContent;
+    const fields = form.fields as FormFields;
+    const applyingWithField = fields.haveDocSigned as FormOptions;
+    expect(applyingWithField.type).toBe('radios');
+    expect(applyingWithField.classes).toBe('govuk-radios');
+    expect((applyingWithField.values[0].label as LanguageLookup)(generatedContent)).toBe(en.one);
+    expect((applyingWithField.values[1].label as LanguageLookup)(generatedContent)).toBe(en.two);
+  });
+
+  test('should contain saveAndComeLater button', () => {
```

Comment:
@monalisasarkar1 Can you please add test case for Continue button ?.

Reply:
done